lib/Gateway.js

```diff
@@ -1091,7 +1091,7 @@ Gateway.prototype.valueTopic = function (node, valueId, returnObject = false) {
  * @param {number} nodeID
  */
 Gateway.prototype.rediscoverNode = function (nodeID) {
-  const node = this.zwave.nodes[nodeID]
+  const node = this.zwave.nodes.get(nodeID)
   if (node) {
```

Comment:
If you check first with `this.zwave.nodes.has(nodeID)`, you shouldn't need to check if `node` is truthy.

Reply:
yeah did mostly find/replace in the code :)